src/components/game-setup/GameStepper.js

```diff
@@ -0,0 +1,137 @@
+import * as React from 'react';
```

Comment:
You don't actually need to import react anymore, instead we should be import which methods we'll use from react.

`import { useState } from 'react';`